src/collection/components/CollectionOrBundleEditAdmin.tsx

```diff
@@ -53,23 +54,22 @@ const CollectionOrBundleEditAdmin: FunctionComponent<CollectionOrBundleEditAdmin
 
 	const updateCollectionMultiProperty = (
 		selectedTagOptions: TagInfo[],
-		collectionProp: keyof Avo.Collection.Collection | 'collection_labels' // TODO remove labels once update to typings v2.14.0
+		collectionProp: keyof Avo.Collection.Collection
 	) => {
 		changeCollectionState({
 			collectionProp,
 			type: 'UPDATE_COLLECTION_PROP',
 			collectionPropValue: (selectedTagOptions || []).map(tag => ({
 				label: tag.value as string,
-			})) as any, // TODO remove cast to any once update to typings v2.14.0
+			})) as any,
```

Comment:
Mag die `as any` dan weg?

Reply:
nee, die {label} objecten zijn slechts een partial collectionLabel object, meer hebben we niet nodig om ze te kunnen opslaan, maar typescript is daar niet mee akkoord, vandaar die cast naar any